runtime/oti/xhelpers.m4

```diff
@@ -26,6 +26,8 @@ ifdef({WIN32},{
 
 ifdef({ASM_J9VM_ENV_DATA64},{
 
+define({SHORT_JMP},{short})
+
```

Comment:
Should this not be outside the 64-bit ifdef?

Reply:
Good point - updated in new commit